src/Weather/Rasp/RaspStyle.cpp

```diff
@@ -135,5 +439,49 @@ const RaspStyle rasp_styles[] = {
     1, // max range 256*(1**2) = 512 cm/s = 5.0 m/s
     false },
   { _T("blcwbase"), rasp_colors[2], 4, false },
+
+  // Thermalmap.info Colorschemes and Rasp file names
+  { _T("ThermalStrength"), rasp_colors[7], 4, false },
+  { _T("ThermalHeight"), rasp_colors[8], 4, false },
+  { _T("Convergence"), rasp_colors[6], 5, false },
+  { _T("Wave_1000m"), rasp_colors[6], 5, false },
+  { _T("Wave_2000m"), rasp_colors[6], 5, false },
+  { _T("Wave_3000m"), rasp_colors[6], 5, false },
+  { _T("Wave_4000m"), rasp_colors[6], 5, false },
+  { _T("Wave_5000m"), rasp_colors[6], 5, false },
+  { _T("Wave_6000m"), rasp_colors[6], 5, false },
+  { _T("Wave_7000m"), rasp_colors[6], 5, false },
+  { _T("Temperature2m"), rasp_colors[9], 4, false },
+  { _T("Rain"), rasp_colors[10], 3, false },
+  { _T("PFD_Day_DuoDiscus"), rasp_colors[11], 3, false },
+  { _T("PFD_Day_LS4"), rasp_colors[11], 3, false },
+  { _T("PFD_Day_K8"), rasp_colors[11], 3, false },
+  { _T("Windspeed10m"), rasp_colors[12], 3, false },
+  { _T("Windspeed_500m"), rasp_colors[12], 3, false },
```

Comment:
Why do all have an underscore, only "10m" not?

Reply:
@Developer66 in your file that you provide, can you please correct the layer name?